From 26a3c99b111a1f4c15cc002bb940089df2498e0d Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 09:38:40 +0700
Subject: [PATCH 249/433] ARM: dts: r8a7742: Add 3DX graphics (PowerVR G6400)
 node

Add definition of 3DX graphics module to the R8A7742 (RZ/G1H) SoC dtsi.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index 6c7141c..94c9112 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -1184,6 +1184,16 @@
 			status = "disabled";
 		};
 
+		sgx@fd000000 {
+			compatible = "renesas,sgx";
+			reg = <0 0xfd000000 0 0x10000>;
+			interrupts = <0 119 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "irq_sgx";
+			clocks = <&mstp1_clks R8A7742_CLK_3DG>;
+			clock-names = "sgx_clk";
+			power-domains = <&sysc R8A7742_PD_RGX>;
+		};
+
 		msiof0: spi@e6e20000 {
 			compatible = "renesas,msiof-r8a7742",
 				     "renesas,rcar-gen2-msiof";
-- 
2.7.4

